### PR TITLE
feat(sdds-acore/uikit): CheckBoxGroup and RadioBoxGroup.

### PR DESCRIPTION
**Pull request by malilex**
CheckBoxGroup and RadioBoxGroup were implemented and added to sandbox.

https://github.com/user-attachments/assets/bfc4ee2b-cad8-46d8-8f9c-5ce4eb31c675

### CODE REVIEW

File: playground/sandbox/src/main/kotlin/com/sdds/playground/sandbox/radiobox/group/RadioBoxGroupFragment.kt [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                radioBoxGroup?.populate(state)
+            }
+            .launchIn(viewLifecycleOwner.lifecycleScope)
+    }
+
+    private fun RadioBoxGroup.populate(state: RadioBoxUiState) {
+        removeAllViews()
+        val theme = ContextThemeWrapper(context, state.variant.styleRes)
+        repeat(3) {
+            val checkBox = RadioBox(theme)
```
Comment (raininforest) [inline playground/sandbox/src/main/kotlin/com/sdds/playground/sandbox/radiobox/group/RadioBoxGroupFragment.kt:67]: radioBox

File: sdds-core/uikit/src/main/kotlin/com/sdds/uikit/CheckBoxGroup.kt [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+        _hasMainCheckBox = typedArray.getBoolean(R.styleable.CheckBoxGroup_sd_hasMain, false)
+        _itemOffset = typedArray.getDimensionPixelSize(R.styleable.CheckBoxGroup_sd_itemOffset, 0)
+        _itemSpacing = typedArray.getDimensionPixelSize(R.styleable.CheckBoxGroup_sd_itemSpacing, 0)
+        typedArray.recycle()
+        super.setOnHierarchyChangeListener(_passThroughListener)
+        orientation = VERTICAL
+    }
+
+    override fun addView(child: View?, index: Int, params: ViewGroup.LayoutParams?) {
+        // First CheckBox always be main if attribute app:sd_hasMain is true
```
[the remark this answers is not in the document]
Comment (malilex) [reply sdds-core/uikit/src/main/kotlin/com/sdds/uikit/CheckBoxGroup.kt:53]: справедливо)

File: playground/sandbox/src/main/kotlin/com/sdds/playground/sandbox/radiobox/group/RadioBoxGroupFragment.kt (hunk 1 above)
Comment (malilex) [reply playground/sandbox/src/main/kotlin/com/sdds/playground/sandbox/radiobox/group/RadioBoxGroupFragment.kt]: поправил

File: sdds-core/uikit/src/main/kotlin/com/sdds/uikit/CheckBoxGroup.kt (hunk 2 above)
[the remark this answers is not in the document]
Comment (malilex) [reply sdds-core/uikit/src/main/kotlin/com/sdds/uikit/CheckBoxGroup.kt]: перевел)

**Review by angirb — APPROVED** [review]

**Review by raininforest — APPROVED** [review]

**Pull request merged**